cibuildwheel/resources/pinned_docker_images.cfg

```
@@ -1,27 +1,27 @@
 [x86_64]
 manylinux1 = quay.io/pypa/manylinux1_x86_64:2021-04-05-14df3a6
```

```suggestion
manylinux1 = quay.io/pypa/manylinux1_x86_64:2021-05-05-b64d921
```
Commit just before `auditwheel==4.0.0`
Should we ?